microsoft-edge/webview2/get-started/winui.md

```diff
@@ -51,195 +51,209 @@ Return here and continue with the steps below.
 
-   ![The New Universal Windows Platform Project dialog with selected values for Target version and Minimum version.](media/winui-getting-started-project-type.png)
-       
-   Solution Explorer displays the two new projects that were generated:
+    ![The new WinUI 3 project in Solution Explorer](winui-images/new-winui3-project.png)
 
-   *  **Your project name (Desktop)**.  The Desktop project contains the code for your app.  The `App.xaml.cs` file defines an `Application` class that represents your app instance. The `MainWindow.xaml.cs` file defines a `MainWindow` class that represents the main window displayed by your app instance.  The classes derive from types in the `Microsoft.UI.Xaml` namespace of WinUI.
+    *  The `App.xaml.cs` file defines an `Application` class that represents your app instance.
 
-   *  **Your project name (Package)**.  The Package project is a Windows Application Packaging Project that is configured to build the app into an MSIX package for deployment. The project contains the package manifest for your app, and is the startup project for your solution by default.  For more information, see [Set up your desktop application for MSIX packaging in Visual Studio](/windows/msix/desktop/desktop-to-uwp-packaging-dot-net) and [Package manifest schema reference for Windows 10](/uwp/schemas/appxpackage/uapmanifestschema/schema-root).
+    *  The `MainWindow.xaml.cs` file defines a `MainWindow` class that represents the main window displayed by your app instance.  The classes derive from types in the `Microsoft.UI.Xaml` namespace of WinUI.
 
-1. In Solution Explorer, open `MainWindow.xaml`.
+1.  Select **File** > **Save All** (`Ctrl`+`Shift`+`S`) to save the project.
 
-1. Select **File** > **Save All** (`Ctrl`+`Shift`+`S`) to save the project.
+1.  In the **Solution Configurations** dropdown list, select **Debug**.
 
-1. Press **F5** to build and run the project.
+1.  In the **Solution Platforms** dropdown list, select a platform, such as **x64**.
 
+1.  Press **F5** to build and run the project.  The blank WinUI Desktop app opens, with no WebView2 control added yet:
 
-<!-- maintenance link; keep: main copy:
-[Install the WebView2 SDK](../how-to/machine-setup.md#install-the-webview2-sdk) in _Set up your Dev environment for WebView2_
--->
-
-<!-- ====================================================================== -->
-## Step 4 - Add a WebView2 control to your project
+    ![The new blank WinUI 3 app](winui-images/blank-winui3-app.png)
 
-The WindowsAppSDK includes the WebView2 SDK and control, and you shouldn't need to separately install the WebView2 SDK. Edit the `MainWindow.xaml` and `MainWindow.xaml.cs` files to a WebView2 control to the sample app, as follows.
+1.  Close the sample app.
 
-1. In Visual Studio, in Solution Explorer, select `MainWindow.xaml` to open it in the code editor.
 
-   Add the WebView2 XAML namespace, as follows:
+#### Updating target version numbers
```

Comment:
User entered an Issue, implying this section is needed in some special case such as an existing project.  If needed, would this section go here?